test/Air/Air.test.js

```diff
@@ -310,20 +310,22 @@ describe('#AirService', () => {
   });
 
   describe('getTicket', () => {
```

Comment:
I'd like you to keep it as it was. I have fix for this test in another branch, because as you admitted in really makes requests.